### ISSUE TITLE
Add Support for Lumaris Tape Light

### ISSUE THREAD

**Issue by RBaragona** [gurumitts/pylutron-caseta#149]
Add support for https://luxury.lutron.com/us/en/lighting/lumaris-tunable-white-tape-light

```
"8157": {
          "device_id": "8157",
          "current_state": 0,
          "fan_speed": null,
          "zone": "8157",
          "name": "Kitchen_Island Toe Kick",
          "button_groups": null,
          "type": "WhiteTune",
          "model": null,
          "serial": null,
          "area": "587",
          "device_name": "Island Toe Kick",
          "tilt": null
```

I attempted to add this myself (https://github.com/RBaragona/pylutron-caseta/tree/Support-Lumaris-Tape-Light), following https://github.com/gurumitts/pylutron-caseta/issues/115#issuecomment-1250413666, but my version continued to downgrade to 0.18.2.  Any advice for adding a development version would be appreciated.

**Comment by mdonoughe** [comment]
I don't think those instructions work. You should install the library into config/deps. Remember to uninstall it later or you'll still be using the config/deps version even when the version included in Home Assistant is newer.

**Comment by RBaragona** [comment]
Thanks for the quick repsonse, @mdonoughe .  Unfortunately, I'm not familiar with that process and couldn't quickly find any documentation.  Can you point to any instructions or steps I can take to install the library to config/deps?

**Comment by mdonoughe** [comment]
I tried writing documentation for it: https://github.com/mdonoughe/developers.home-assistant/blob/lib-test/docs/api_lib_testing.md

**Comment by RBaragona** [comment]
Thanks for this write up, it’s very helpful!  I think the issue I’m having is with wheel compilation as you’ve stated. I can compile/install if I don’t specify the does folder. Once I do specify deps folder, compiling fails for cffi. I’ll need to do some more research in that direction.

**Comment by RBaragona** [comment]
Finally had enough time to dig into getting this setup, per @mdonoughe 's instructions, I wasn't able to cross compile in to production setup.  I needed the time to learn how to setup a disposable Home Assistant installation by following the [development environment instructions](https://developers.home-assistant.io/docs/development_environment).

**Comment by mdonoughe** [comment]
Fixed in #154
